docs/configuration/elements.md

```diff
@@ -82,6 +82,7 @@ Global arc options: `Chart.defaults.global.elements.arc`.
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
+| `angle` | `number` | `circumference / (arc count)` - for polar only | Arc angle to cover.
```

Comment:
what does "for polar only" mean? That the option is available only for the polar chart or that default is `circumference / (arc count)` only for the polar chart? If the latter, what is the default for other chart types?

Reply:
We have two types of chart using elements.arc : 
 * Polar area
 * Pie/Doughnut

I tried to set `elements.arc.angle`option in Pie/Doughnut chart. It doesn't override the natural angles computation.

So it seems that this option is available only for polar chart, and I think it's a good thing.

Reply:
Ok. I think it might be clearer to put "for polar only" next to the option name rather than the default value. How about: 

`angle` (polar area chart only)

Reply:
Rather than stating its for polar only, why not move it to https://github.com/chartjs/Chart.js/blob/f68b8adaac713f4ce321f4f53fdcd319e021b584/docs/charts/polar.md#L58-L68

Like `weight` in pie/doughnut